Sources/App/githubRoutes.swift

```diff
@@ -0,0 +1,59 @@
+import Vapor
+
+public func githubRoutes(router: GithubCommandRouter) throws {
+    router.register(command: "@vapor-bot", "test performance") { (req) -> Future<HTTPStatus> in
```

Comment:
Like discussed in Discord, would be nice to decode the Webhook data before entering the closure, and default to the name of the bot as first argument. So you'd have this:
```suggestion
    router.command(command: "test performance") { (webhookData) -> Future<HTTPStatus> in
```

Reply:
I think decoding the webhook is a good idea, however I'm going to hold off on using the bot's name as a default. There are situations where we'll want to use the bot without it being called. For example, Tanner suggested that it would be helpful to suggest folks edit their comments to 

    ```swift
    ...
    ```

when they post something like
\`\`\`
...
\`\`\`

It would be really nice to have route groups available in order to handle the fact that there will be so many commands starting with "@vapor-bot"

```swift
let tagCommand = router.grouped("@vapor-bot")

tagCommand.command(command: "test performance") 
```

Reply:
Ah, I can see where you're coming from.

In the "bot world" so to say, you'd differentiate the two as Commands and Listeners where a command is a response to an actual command like
```
@vapor-bot test this
```
and a listener "listens" to certain events. For example on Discord you'd listen to a `MessageCreate` event to listen for new created messages, or a `ReactionAdd` event to listen for added reactions.
A listener is where you'd put logic regarding the codeblocks in this case

Reply:
I would however, advise against doing that now, since it requires a lot more work structuring the GH API interaction. So "grouping" so to say, commands together would be a great interim solution

Reply:
I'm going to hold off on command grouping until a future PR. The webhooks are being decoded and passed to the routing closure now.